test/integration/test.js

```diff
@@ -375,16 +381,19 @@ describe('integration', function() {
         },
         function() {
           var ravenData = iframe.contentWindow.ravenData[0];
-          assert.match(ravenData.exception.values[0].type, /^Error/);
+          // IE10 and Edge14 ¯\_(ツ)_/¯
+          assert.match(ravenData.exception.values[0].type, /^Error|undefined/);
```

Comment:
Same here as above.

Reply:
This was my initial concern and that's why I added that check in the first place. Apparently checking for Edge version using `userAgent` is trivial.